sys/net/gnrc/link_layer/gomach/gomach_internal.c

```diff
@@ -488,7 +488,13 @@ int gnrc_gomach_send_beacon(gnrc_netif_t *netif)
                 total_tdma_slot_num -= redueced_slots_num;
                 break;
             }
+
```

Comment:
Any reasons for these newlines?

Reply:
Em, just to keep a space between the above block and the following counter increasement.

Reply:
The `j++;` looks a little lonely now, but okay.

Reply:
OK, will be careful on this next time (not add new lines). :-)